src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -206,6 +206,10 @@ message PageStreamingConfigProto {
 message PageStreamingRequestProto {
   // The name of the field in the request containing the page token.
   string token_field = 1;
+
+  // The name of the field in the request containing the desired number of items
+  // in the next page.
+  string page_size_field = 2;
```

Comment:
Optional: changes to the GAPIC yaml should also involve a change to the config generator. I've already done that in #147, so it should be OK if that change follows this one.